packages/net-stubbing/lib/server/util.ts

```diff
@@ -145,7 +145,7 @@ export async function setResponseFromFixture (getFixtureFn: GetFixtureFn, static
  * @param backendRequest BackendRequest object.
  * @param staticResponse BackendStaticResponse object.
  */
-export function sendStaticResponse (backendRequest: BackendRequest, staticResponse: BackendStaticResponse) {
+export function sendStaticResponse (backendRequest: Pick<BackendRequest, 'onError' | 'onResponse'>, staticResponse: BackendStaticResponse) {
```

Comment:
what's this change?

Reply:
allows this to work since the type overlaps with BackendRequest https://github.com/cypress-io/cypress/blob/f62562a22193c0258f4e05654b5e63d9cc6b37fc/packages/net-stubbing/lib/server/intercept-request.ts#L36-L46